auth-api/src/auth_api/resources/v1/org.py

```diff
@@ -182,7 +182,7 @@ def put_organization(org_id):
     if not valid_format:
         return {"message": schema_utils.serialize(errors)}, HTTPStatus.BAD_REQUEST
     try:
-        org = OrgService.find_by_org_id(org_id, allowed_roles=(*CLIENT_ADMIN_ROLES, STAFF))
+        org = OrgService.find_by_org_id(org_id, allowed_roles=(*CLIENT_ADMIN_ROLES, STAFF, USER))
```

Comment:
You're giving a user the ability to edit an org now? Shouldn't it just be the contact info?

Reply:
Otherwise users can update the payment info

```
        if name_updated or payment_info:
            Org._create_payment_for_org(mailing_address, self._model, payment_info, False)

```

I think you just want this code (double check)

```
        # Update mailing address Or create new one
        if mailing_address:
            has_org_updates = True
            contacts = self._model.contacts
            if len(contacts) > 0:
                contact = self._model.contacts[0].contact
                contact.update_from_dict(**camelback2snake(mailing_address))
                contact.save()
            else:
                Org.add_contact_to_org(mailing_address, self._model)


        if has_org_updates:
            excluded = ("type_code",) if has_status_changing else EXCLUDED_FIELDS
            self._model.update_org_from_dict(camelback2snake(org_info), exclude=excluded)
            if is_govm_account_creation:
                # send mail after the org is committed to DB
                Org.send_staff_review_account_reminder(relationship_id=self._model.id)


        Org._publish_activity_on_mailing_address_change(org_model.id, current_org_name, mailing_address)
```